config/Config-template.groovy

```diff
@@ -251,6 +271,16 @@ environments {
 }
 /* }}} */
 
+/* {{{ GWAS Configuration */
+
+com.recomdata.dataUpload.appTitle="Upload data to tranSMART"
+//com.recomdata.dataUpload.templates.dir="/home/transmart/templates/"
+//com.recomdata.dataUpload.uploads.dir="/home/transmart/uploads/"
+//com.recomdata.dataUpload.etl.dir="/home/transmart/ETL/"
+com.recomdata.dataUpload.stageScript="run_analysis_stage"
```

Comment:
Better not to put commented out options here and include them instead in `Config-extra.php.sample`, because that's the file people are most likely to look at.

Additionally, it would be nice to make this developer friendly and have defaults that work fine in development mode. For the directories, you could use the same strategy as `https://github.com/transmart/transmart-data/blob/master/config/Config-template.groovy#L579`. For the location of `tranSMART-ETL`, it's more complicated, but `build-config.groovy` could be changed to replaced some placeholders (e.g. `@@KETTLE_JOBS_ORA@@`) with the value of the corresponding environment variable.